### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by AlexFreik**

### CODE REVIEW

File: docs/docs/we-are-hiring.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## How we work
+![Our Dream team](https://user-images.githubusercontent.com/61039123/188391737-fccd412a-bba0-48b2-87c0-ed859c89eedb.jpeg){ width="50%" align="right" }
+
+- **Ambition shapes reality**: We are extremely ambitious in what we can accomplish. We set high standards, expecting to fail in the short-term but knowing that failure guides us to learn and ultimately succeed.
+- **Learn through the building**: It's time to build! We ship frequently, connect with our community, and iterate until we get it perfect.
+
+## Who are we?
+Our team (right now) consist of two members. We are
+- a programmer with a background in Physics and 
+- the hurt of the project - a businessman with a programming background.
```
Comment (roman-timo) [inline docs/docs/we-are-hiring.md:21]: I think this docs is fantastic, except maybe only this one thing. I don't fully get what did you mean by "the hurt of the project" O_o Ahahhahha))) This phrase gonna make me hurt  hahahaha))) Maybe I should fix this businessman so he is not a hurt for our project anymore. Will talk to him! )))hahah)))

**Review by roman-timo — CHANGES_REQUESTED** [review]

File: docs/docs/we-are-hiring.md (hunk 1 above)
Comment (AlexFreik) [reply docs/docs/we-are-hiring.md]: ahah) true

Comment (AlexFreik) [reply docs/docs/we-are-hiring.md]: fixed, thaks)

**Review by roman-timo — APPROVED** [review]
all good

**Pull request merged**